boltz_client/cli.py

```diff
@@ -5,20 +5,20 @@
 
 import click
 
-from boltz_client.boltz import BoltzClient, BoltzConfig
+from boltz_client.boltz import BoltzClient, BoltzConfig, SwapDirection
 
 # disable tracebacks on exceptions
 sys.tracebacklimit = 0
 
 config = BoltzConfig()
 
 # use for manual testing
-# config = BoltzConfig(
-#     network="regtest",
-#     api_url="http://localhost:9001",
-#     mempool_url="http://localhost:8999/api/v1",
-#     mempool_ws_url="ws://localhost:8999/api/v1/ws",
-# )
+config = BoltzConfig(
+    network="regtest",
+    api_url="http://localhost:9001",
+    mempool_url="http://localhost:8999/api/v1",
+    mempool_ws_url="ws://localhost:8999/api/v1/ws",
+)
```

Comment:
That was in a comment before. In general, this seems like a pretty horrible workflow